composer.json

```diff
@@ -118,6 +124,46 @@
       "@auto-scripts"
     ]
   },
+  "archive": {
+    "exclude": [
```

Comment:
I do not see how this would work elegantly. This could be achieved by excluding everything, and then including stuff with a `!` prefix. In my opinion that would only be marginally better. Yes it decreases the number of additions to the archive excludes list, but makes the list harder to grasp in my opinion.

Reply:
In perspective of defence-in-depth it would be very valuable to only include needed files in a release. This would also prevent unexpected file additions and it will cost less time to maintain.